angular-site/src/app/data-table/sort-header/sort-overview-example.ts

```diff
@@ -0,0 +1,58 @@
+import {Component} from '@angular/core';
+import {Sort} from '@angular/material/sort';
+
+export interface Dessert {
+  calories: number;
+  carbs: number;
+  fat: number;
+  name: string;
+  protein: number;
+}
+
+/**
+ * @title Sorting overview
+ */
+@Component({
+  selector: 'sort-overview-example',
+  templateUrl: 'sort-overview-example.html',
+  styleUrls: ['sort-overview-example.css'],
+})
+export class SortOverviewExample {
```

Comment:
new line , and so on formatting issues in another places. try to use js code style formatter (you can start from here https://habr.com/ru/post/220229/ I guess)

Reply:
According to the [Angular coding style guide](https://angular.io/guide/styleguide) new line is not needed here. Do you think I should add it nevertheless?